### ISSUE TITLE
Удаление администратора

### ISSUE THREAD

**Issue by kirzas** [sedovalx/taxi#36]
Сейчас можно удалить текущего пользователя, т.е. самого себя. Нужно это запретить в виде правила валидации как на сервере, так и на клиенте.
- Удалил самого себя через UI
- Пользователь удалился. Залогинится больше нельзя.
- Не должно быть возможности удалять текущего пользователя.

**Comment by sedovalx** [comment]
Это вопрос? Что конкретно ломается?

**Comment by kirzas** [comment]
это не вопрос :) просто тут на маке я не знаю, где запятая
суть в том, что через интерфейс можно удалить пользователя админ/админ, даже если он единственный пользователь. если его удалить, то залогиниться нельзя.
@sedovalx

**Comment by sedovalx** [comment]
Понял. Это баг, но пока некритичный. Зависит от #19 и #34.

**Comment by kirzas** [comment]
@sedovalx 
я правильно сделал, что создал issue?

**Comment by sedovalx** [comment]
@Argelein да, конечно - все баги нужно фиксировать в виде issue с меткой Bug. Только вот описывать их нужно более конкретно:
- Что делал
- Что получил
- Что должен был получить

**Comment by sedovalx** [comment]
Сервер теперь запрещает удалять самого себя, однако на клиенте запись в списке все равно пропадает и появляется только при следующей перезагрузке списка. Нужно либо как-то восстанавливать запись при ошибке, либо запрещать на клиенте удалять самого себя.
